src/Auth/EmailVerification/EmailVerificationRepository.php

```diff
@@ -0,0 +1,206 @@
+     *
+     * @var \Illuminate\Database\Eloquent\Model
+     */
+    protected $authModel;
+
+    /**
+     * Constructor
+     *
+     * @author Paulius Navickas <pana@nodes.dk>
+     *
+     * @access public
+     * @param  \Nodes\Api\Auth\EmailVerification\EmailVerificationModel $model
+     * @throws \Nodes\Api\Auth\Exceptions\MissingUserModelException
+     */
+    public function __construct(EmailVerificationModel $model)
+    {
+        $this->setupRepository($model);
+
+        // Retrieve namespace of API auth model
+        $userModel = config('nodes.api.auth.model', null);
+        if (empty($userModel)) {
+            throw new MissingUserModelException('No API auth model has been specified');
+        }
+
+        // Instantiate auth model
+        $this->authModel = app($userModel);
+    }
+
+    /**
+     * Retrieve by token
+     *
+     * @author Paulius Navickas <pana@nodes.dk>
+     *
+     * @access public
+     * @param  string $token
+     * @return \Nodes\Api\Auth\EmailVerification\Model
+     */
+    public function getByToken($token)
+    {
+        dd($token);
+        return $this->where('token', '=', $token)
+            ->first();
+    }
+
+    /**
+     * Retrieve by unexpired token
+     *
+     * @author Paulius Navickas <pana@nodes.dk>
+     *
+     * @access public
+     * @param  string $token
+     * @return \Nodes\Api\Auth\ResetPassword\ResetPasswordModel
+     */
+    public function getByUnexpiredToken($token)
+    {
+        return $this->where('token', '=', $token)
+            ->where('expire_at', '>', Carbon::now()->format('Y-m-d H:i:s'))
+            ->first();
+    }
+
+    /**
+     * Generate and send a email with link instructions
+     *
+     * @author Paulius Navickas <pana@nodes.dk>
+     *
+     * @access public
+     * @param  array $conditions WHERE conditions to locate user. Format: ['column' => 'value']
+     * @return boolean
+     * @throws \Nodes\Api\Auth\Exceptions\ResetPasswordNoUserException
+     */
+    public function sendVerificationEmail($user)
+    {
+        if (empty($user)) {
+            throw new ResetPasswordNoUserException('Could not find any user with those credentials');
+        }
+
+        // Generate email verification token
+        $token = $this->generateEmailVerificationToken($user);
+
+        // Generate email verification domain
+        $domain = env('NODES_ENV', false) ? sprintf('https://%s.%s', env('APP_NAME'), env('APP_DOMAIN')) : config('app.url');
+
+        // Send e-mail with instructions on how to reset password
+        $status = (bool) Mail::send([
+            'html' => config('nodes.api.email-verification.views.html', 'nodes.api::email-verification.emails.html'),
+            'text' => config('nodes.api.email-verification.views.text', 'nodes.api::email-verification.emails.text')
+        ], [
+            'user' => $user,
+            'domain' => $domain,
+            'email' => $user->email,
+            'token' => $token,
```

Comment:
Using incorrect field since you've renamed the field from `token` to `verification`